src/Security/src/Microsoft.AspNetCore.Authentication.OpenIdConnect/ClientAuthentication/EncodingHelper.cs

```diff
@@ -0,0 +1,13 @@
+using System.Text;
+
+namespace Microsoft.AspNetCore.Authentication.OpenIdConnect
+{
+    internal static class EncodingHelper
+    {
+        internal static string ToUTF8(this string text)
```

Comment:
I'm not sure what the point of this is. The string is utf-16 before and after. At best some characters would get filtered out, or at worst some would throw.

Reply:
look back at the original PR for this

Reply:
There's no explanation in the other PR. If it's supposed to have a side effect then add a test that demonstrates it.